x-pack/plugins/security_solution/public/management/pages/policy/view/policy_forms/protections/malware.tsx

```diff
@@ -189,14 +191,17 @@ export const MalwareProtections = React.memo(() => {
             />
           </h6>
         </EuiTitle>
+        <EuiText color="subdued" size="xs">
+          <i>{`Endpoint versions ${popupVersionsMap.get('malware')}`}</i>
```

Comment:
this needs to be i18n'd (I think maybe you had this in the `map` file at one point, then moved it here)

Reply:
yeah okie that makes sense